addons/medical_treatment/stringtable.xml

```diff
@@ -55,9 +62,11 @@
             </Key>
             <Key ID="STR_ACE_Medical_treatment_allowSelfIV_DisplayName">
                 <English>Self IV Transfusion</English>
+                <German>Eigennutzung von Bluttransfusionen</German>
             </Key>
             <Key ID="STR_ACE_Medical_treatment_allowSelfIV_Description">
                 <English>Allows using IV Transfusions on yourself</English>
+                <German>Erlaube Bluttransfusionen bei sich selbst zu nutzen</German>
```

Comment:
"an sich selbst" ?

Reply:
Suggestion? :D @dedmen

Reply:
Yeah..
I think "an sich selbst benutzen" is more correct.
"bei" is just next to yourself, not "on yourself".